testing/insulating_clients.rst

```diff
@@ -8,35 +8,56 @@ If you need to simulate an interaction between different clients (think of a
 chat for instance), create several clients::
 
     // ...
+    use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
     use Symfony\Component\HttpFoundation\Response;
 
-    $harry = static::createClient();
-    $sally = static::createClient();
+    class MyTest extends WebTestCase
+    {
+        // ...
+        public function testWithMultipleClients()
+        {
+            self::ensureKernelShutdown();
```

Comment:
This first shutdown is not really needed, no?

Reply:
This part of the documentation is for multiple clients, I thought it would be better to have some kind of patter and add the call prior to all client creations (which is not present on the examples of a single client) to show the main difference and easy the understanding of the mechanism.

But I am happy with it if you want me to remove it and/or add a comment about the need to add it only if there are new clients being create.

Reply:
yeah, I opened #34507 a long time ago and proposed this solution on the docs. To be honest, feels weird to ask people to shut down the kernel before actually creating any client. And when you call `createClient()` it is already shutting down any previous kernel on the `bootKernel()` method.

I'll leave it up to the maintainers!